iphone/Classes/TiUISearchBar.m

```diff
@@ -118,31 +118,44 @@ - (void)setHintText_:(id)value
   [[self searchBar] setPlaceholder:[TiUtils stringValue:value]];
 
   if ([[self proxy] valueForUndefinedKey:@"hintTextColor"]) {
-    [self setHintTextColor_:[[self proxy] valueForUndefinedKey:@"hintTextColor"]];
+    if ([TiUtils isIOSVersionOrGreater:@"13.0"]) {
+      // Need to call a bit later to get searchTextField loaded
+      [self performSelector:@selector(setHintTextColor_:) withObject:[[self proxy] valueForUndefinedKey:@"hintTextColor"] afterDelay:.01];
```

Comment:
Do you have more info why this is required? I'd love to know why this only works after adding the delay.

Reply:
In debugging I found that SearchTextField inside searchbar takes sometime to load and before loading the searchTextField  setting this property is not working.